pkg/storage/fs/ocis/node.go

```diff
@@ -273,32 +277,60 @@ func (n *Node) Parent() (p *Node, err error) {
 
 // Owner returns the cached owner id or reads it from the extended attributes
 // TODO can be private as only the AsResourceInfo uses it
-func (n *Node) Owner() (id string, idp string, err error) {
-	if n.ownerID != "" && n.ownerIDP != "" {
-		return n.ownerID, n.ownerIDP, nil
+func (n *Node) Owner() (o *userpb.UserId, err error) {
```

Comment:
Am I blind or is the UserId param never being used?

Reply:
it is returned ;-)